ebos/eclwriter.hh

```diff
@@ -507,6 +515,7 @@ private:
     void prepareLocalCellData(const bool isSubStep,
                               const int  reportStepNum)
     {
+        if( !(eclOutputModule_->localDataValid()) ){
```

Comment:
You might alternatively write this as
```C++
if (eclOutputModule_->localDataValid()) {
    return;
}
```
which would save one level of nesting.

Reply:
changed it.